.github/workflows/nomad.yaml

```diff
@@ -303,8 +307,15 @@ jobs:
           keep_files: true
           enable_jekyll: true
 
-      - name: Upload run summary
-        uses: actions/upload-artifact@v6
-        with:
-          name: summariser-report
-          path: ./summariser-report-*.json
+      - name: Upload run summary to bucket
+        env:
+          AWS_ACCESS_KEY_ID: ${{ secrets.HETZNER_HOLOCHAIN_INFRA_BUCKETS_ACCESS }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.HETZNER_HOLOCHAIN_INFRA_BUCKETS_SECRET }}
+          AWS_DEFAULT_REGION: fsn1
```

Comment:
What's this region?

Reply:
EU Central. See https://docs.hetzner.com/cloud/general/locations/